joomla/joomla-cms
# Add Unpublished Content Language on Language Package Install

Pull Request for Issue https://github.com/joomla/joomla-cms/issues/11376.
### Summary of Changes

Adds a new (unpublished) content language on language package install.

After installing all languages packs ...
![languages](https://cloud.githubusercontent.com/assets/9630530/18149992/b6fb22b6-6fdb-11e6-9ac8-90ffdd01e44b.gif)

Note: this is a RFC, it still have issues in the Joomla multilingual installation process (that already adds a content language).
### Testing Instructions
1. Apply patch
2. Try to install several language packages (ex: fr-FR, it-IT) and also several in the same language (en-AU, en-US, etc)
3. Check all content languages are created for the languages packages you installed
4. Code review.
### Documentation Changes Required

None.
